src/main/scala/com/github/anicolaspp/spark/MapRDB.scala

```diff
@@ -19,22 +21,22 @@ object MapRDB {
   }
 
   implicit class ExtendedDataFrame(df: DataFrame) {
-    def saveToMapRDB(path: String, withTransaction: Boolean = false): Unit = {
+    def writeToMapRDB(path: String, withTransaction: Boolean = false): Unit = {
 
       if (withTransaction) {
         df.write
           .format("com.github.anicolaspp.spark.sql.writing.Writer")
           .save(path)
 
       } else {
+        MapRSpark.save(df, path, "_id", false, false)
```

Comment:
If we want the connector to replace the official connector at some point then the `createTable` and `bulkInsert` options should be exposed to the caller. Either through default parameters or through the .option method. Similar to how the current connector allows `df.write.option("Operation","Insert").saveToMapRDB("path")`

Reply:
That is fine. I will create an issue to track that. 
GH-9